### ISSUE TITLE
Colorbar position in subplots

### ISSUE THREAD

**Issue by TheDimLebowski** [ropensci/plotly#492]
In a subplot with 2 colormaps, the colorbars are positioned to the right of the plot.
Code:
```
z1 <- matrix(rexp(200, rate=.1), ncol=20) 
z2 <- matrix(rexp(200, rate=.1), ncol=20) 
p <- subplot( 
    plot_ly(z = z1, type = "heatmap"), 
    plot_ly(z = z2, type = "heatmap"), 
    margin = 0.1 
) %>% embed_notebook(width = 1000, height = 450)
```

![image](https://cloud.githubusercontent.com/assets/6419212/13667293/89b13460-e6ad-11e5-8c67-5ae75cd1b82a.png)

**Comment by cpsievert** [comment]
You can move the `x` position of the first colorbar like so:

```r
l <- plotly_build(p)
l$data[[1]]$colorbar$x <- 0.42
l
```

You really should be able to achieve the same effect by doing this, but it currently isn't working.

```r
z1 <- matrix(rexp(200, rate=.1), ncol=20) 
z2 <- matrix(rexp(200, rate=.1), ncol=20) 
p <- subplot( 
  plot_ly(z = z1, type = "heatmap", list(colorbar = list(x = 0.42))), 
  plot_ly(z = z2, type = "heatmap"), 
  margin = 0.1 
)
```

**Comment by cpsievert** [comment]
Closed via d2bf2a1eb56d824a091e2cc9afea09a5968dd050.
